server/core/server-core_test.go

```diff
@@ -33,7 +33,7 @@ var (
 func TestValidateMessage(t *testing.T) {
 
 	u := url.URL{Scheme: "ws", Host: *addr, Path: "/message"}
-	log.Printf("connecting to %s", u.String())
+	fmt.Println("connecting to %s", u.String())
```

Comment:
`%s` will not be replaced when you using `fmt.Println`.
You should use `t.Logf()` or `fmt.Printf()`